connections/src/app/core/services/qr-code-generator.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { QrCodeGeneratorService } from './qr-code-generator.service';
+
+describe('QrCodeGeneratorService', () => {
+  let service: QrCodeGeneratorService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(QrCodeGeneratorService);
+  });
+
+  it('should be created', () => {
```

Comment:
Lets start adding....
`it('should be created', () => {`
turn to 
`it('should be created, QrCodeGeneratorService', () => {`
so the name of the test, the component, or service shows up
QrCodeGeneratorService

Reply:
Yeah will look into this